test_run.sh

```diff
@@ -112,6 +124,21 @@ xmllint_test() {
   find -L ./bazel-testlogs -iname "*.xml" | xargs -n1 xmllint > /dev/null
 }
 
+multiple_junit_suffixes() {
+  bazel test //test:JunitMultipleSuffixes
+  matches=$(grep -c -e 'Running E2E' -e 'Running IT' ./bazel-testlogs/test/JunitMultipleSuffixes/test.log)
+  if [ $matches -eq 2 ]; then
+    return 0
+  else
+    return 1
+  fi
+}
+
+junit_generates_xml_logs() {
+  bazel test //test:JunitTestWithDeps
+  test -e ./bazel-testlogs/test/JunitTestWithDeps/test.xml
```

Comment:
yeah, just double checked `xmllint` fails on an empty file. Disregard this comment.